apps/web/composables/useModernImage/__tests__/useModernImage.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { mockNuxtImport } from "@nuxt/test-utils/runtime";
+import { useModernImage } from "../useModernImage";
+
+const { useRuntimeConfig } = vi.hoisted(() => {
+    return {
+        useRuntimeConfig: vi.fn().mockImplementation(() => {
+            return {
+                public: {
+                    useAvif: true,
+                    useWebp: true
+                }
+            }
+        })
+    }
+});
+
+// mockNuxtImport can only be called once per test file
```

Comment:
**question**: I understand that this is useful information, but can we maybe move it somewhere else? I don't think we'll want to have this comment in every test file.

Reply:
ok :)